controllers/glance_controller.go

```diff
@@ -396,6 +400,9 @@ func (r *GlanceReconciler) reconcileInit(
 	}
 	instance.Status.Conditions.MarkTrue(condition.DBSyncReadyCondition, condition.DBSyncReadyMessage)
 
+	// when job passed, mark NetworkAttachmentsReadyCondition ready
+	instance.Status.Conditions.MarkTrue(condition.NetworkAttachmentsReadyCondition, condition.NetworkAttachmentsReadyMessage)
+
```

Comment:
I'm confused as to why `NetworkAttachmentsReadyCondition` is set to `True` here and then is potentially set to something else between L495-515.  Should we maybe be doing something like you've proposed in the Keystone changes [1]?

[1] https://github.com/openstack-k8s-operators/keystone-operator/blob/2266fea2a2b862bf77f2aa139b19b94cd228007a/controllers/keystoneapi_controller.go#L634-L652

Reply:
Nevermind, GitHub code collapsing hid the fact that this is a function called later :)